.tool-versions

```diff
@@ -13,3 +13,6 @@ shellcheck 0.9.0
 
 # GUI client
 pnpm 10.18.3
+
+# Android app
+java openjdk-17
```

Comment:
Why are polluting the shared `.tool-versions` ? wouldn't it be better to put it in `android` directory? then the shared tools are shared and the Android specific (JDK) can be specified for people wanting to build android